sklearn/tests/test_pipeline.py

```diff
@@ -1114,6 +1139,32 @@ def test_make_pipeline_memory():
     shutil.rmtree(cachedir)
 
 
+def test_features_names_passthrough():
+    pipe = Pipeline(steps=[
+        ('imputer', 'passthrough'),
+        ('scaler', StandardScaler()),
+        ('select', 'passthrough'),
```

Comment:
I would find it more interesting to have a `SelectKBest` with `k=2` for this test, no?